ITEM.tsv

```diff
@@ -3441,7 +3441,7 @@ ITEM_20150317_003440	Sapper Master's Introduction Letter
 ITEM_20150317_003441	{memo X}Letter of introduction that Sapper Master wrote to the Highlander Master 
 ITEM_20150317_003442	Highlander Master's Proof 
 ITEM_20150317_003443	{memo X}Confirmation from the Highlander Master that you accomplished all of the necessary requirements for advancement. 
-ITEM_20150317_003444	Tanu's Purifying Stone
+ITEM_20150317_003444	Tanu's Purifying Stone 
```

Comment:
I've encountered many unnecessary spaces in item names.

e.g. Tanu's Purifying Stone  (0/1) 

compared to

Tanu's Purifying Stone (0/1)